ui/app/components/app/selected-provider-indicator/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export default class SelectedProviderIndicator extends Component {
+  static contextTypes = {
+    t: PropTypes.func,
+  }
+
+  static propTypes = {
+    isUnlocked: PropTypes.bool,
+  }
+
+  state = {
+    isCurrentProvider: true,
+  }
+
+  componentDidMount () {
+    chrome.braveWallet.getWeb3Provider((providerId) => { // eslint-disable-line no-undef
```

Comment:
It might actually be good to feature check for `getWeb3Provider` since this was only added in 1.16.x.  Some older browsers might get the update.

Reply:
Good thought, done 👍